### PR TITLE
Fix Geneve dissection

### PR DESCRIPTION
**Pull request by inthefog**
Fix Geneve dissection.

With current implementation, in case of multiple Geneve options, only the first option is parsed correctly and the rest options are interpreted as its raw payload.

### CODE REVIEW

**Comment by evverx** [comment]
@inthefog could you remove the `.DS_Store` files from the commit? My guess would be that they ended up there accidentally. (I think it would probably make sense to add `.DS_Store` to `.gitignore`).

Could you also add a test to https://github.com/secdev/scapy/blob/master/test/contrib/geneve.uts to make sure that multiple options are dissected correctly? I don't think this scenario is covered by any tests at this point.

**Comment by inthefog** [comment]
> @inthefog could you remove the `.DS_Store` files from the commit? My guess would be that they ended up there accidentally. (I think it would probably make sense to add `.DS_Store` to `.gitignore`).
> 
> Could you also add a test to https://github.com/secdev/scapy/blob/master/test/contrib/geneve.uts to make sure that multiple options are dissected correctly? I don't think this scenario is covered by any tests at this point.

Done. Thanks for the review.

File: .gitignore [hunk 1]
```diff
@@ -11,7 +11,9 @@ coverage.xml
 .ipynb_checkpoints
 .mypy_cache
 .vscode
+.DS_Store
 [.]venv/
 __pycache__/
 doc/scapy/_build
 doc/scapy/api
+scapy/.DS_Store
```
Comment (evverx) [inline .gitignore:19]: I think this can be dropped. `.DS_Store` (without slashes) should already match at any level (including `scapy/`).

Comment (inthefog) [reply .gitignore]: Done.

Thank you.

**Review by gpotter2 — APPROVED** [review]

**Pull request merged**